Automattic/jetpack
# VideoPress: Fix app breaking when deleting last page video.

Closes #27772 

## Proposed changes:

* Allow pushing `page` changes into store without pushing back to URL page.
* Reduce the page on `REMOVE_VIDEO` action if needed.

### Other information:


## Jetpack product discussion


N/A

## Does this pull request change what data or activity we track or use?


No.

## Testing instructions:






* Spawn a JN site with `VideoPress`.
* Upload 7 videos to have a second page.
* Delete the second-page video.
* You should be redirected to the first page.
* _Try the same with more pages_
